jsf-showcase-webapp/pom.xml

```diff
@@ -137,6 +124,18 @@
 			</dependencies>
 		</profile>
 		<profile>
+			<id>liferay</id>
```

Comment:
@pcwhite, since we test liferay and pluto from bridge-impl now, I think we can just remove these. Please remove them and do a quick retest of webapp with the default browser. Thanks!